sdks/nodejs/src/agonesSDK.js

```diff
@@ -0,0 +1,101 @@
+const grpc = require('grpc');
+
+const messages = require('../lib/sdk_pb');
+const services = require('../lib/sdk_grpc_pb');
+
+class AgonesSDK {
+	constructor() {
+		this.client = new services.SDKClient('localhost:50051', grpc.credentials.createInsecure());
```

Comment:
shouldn't the port be 59357 ?

Reply:
Doh! That explains why I had to keep changing it to test. The port in the example code I was using is 50051. Changed in next commit